forge-gui/res/cardsfolder/upcoming/headliner_scarlett.txt

```diff
@@ -0,0 +1,14 @@
+Name:Headliner Scarlett
+ManaCost:3 R
+Types:Legendary Creature Human Warlock
+PT:3/3
+K:Haste
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Destination$ Battlefield | Execute$ TrigEffect | TriggerDescription$ When CARDNAME enters the battlefield, creatures target player controls can't block this turn.
+SVar:TrigEffect:DB$ Effect | ValidTgts$ Player | TgtPrompt$ Select target player | AILogic$ Evasion | StaticAbilities$ OppCantBlock | RememberObjects$ Targeted
+SVar:OppCantBlock:Mode$ CantBlockBy | ValidBlocker$ Creature.RememberedPlayerCtrl | Description$ Creatures targeted player controls can't block this turn.
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ TrigExile | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of your upkeep, exile the top card of your library face down. You may look at and play that card this turn.
+SVar:TrigExile:DB$ ChangeZone | Defined$ TopOfLibrary | Origin$ Library |  RememberChanged$ True | Destination$ Exile | ExileFaceDown$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | RememberObjects$ Remembered | StaticAbilities$ STPlay | ForgetOnMoved$ Exile | SubAbility$ DBCleanup
+SVar:STPlay:Mode$ Continuous | MayLookAt$ You | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Exile | Description$ You may look at and play that card this turn.
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+Oracle:Haste\nWhen Headliner Scarlett enters the battlefield, creatures target player controls can't block this turn.\nAt the beginning of your upkeep, exile the top card of your library face down. You may look at and play that card this turn.
```

Comment:
don't forget your whitespace lines

Reply:
![image](https://github.com/Card-Forge/forge/assets/67333662/c4e868a4-7a6b-45ca-9355-db549f2db758)

I think this PR is fine ?

Reply:
Sounds good, the github interface makes it hard to tell ;)